### PR TITLE
[v1.3] Add CheckInCommandParserTest

### PR DESCRIPTION
**Pull request by adamwth**
### Test
- Add CheckInCommandParserTest
- Update CheckOutCommandParserTest with more boundary value test cases

### CODE REVIEW

File: src/test/java/seedu/address/logic/parser/CheckoutCommandParserTest.java [hunk 1]
```diff
@@ -28,19 +29,30 @@
...
+                String.format(MESSAGE_INVALID_COMMAND_FORMAT, CheckoutCommand.MESSAGE_USAGE));
         assertParseFailure(parser, ROOM_DESC_001 + DATE_START_DESC_AMY,
                 String.format(MESSAGE_INVALID_COMMAND_FORMAT, CheckoutCommand.MESSAGE_USAGE));
+        assertParseFailure(parser, ROOM_DESC_001 + DATE_END_DESC_AMY,
+                String.format(MESSAGE_INVALID_COMMAND_FORMAT, CheckoutCommand.MESSAGE_USAGE));
+        assertParseFailure(parser, DATE_START_DESC_AMY + DATE_END_DESC_AMY,
+                String.format(MESSAGE_INVALID_COMMAND_FORMAT, CheckoutCommand.MESSAGE_USAGE));
         assertParseFailure(parser, " r/001 from/01/01/18 to/30/02/18",
                 String.format(MESSAGE_INVALID_COMMAND_FORMAT, CheckoutCommand.MESSAGE_USAGE));
+        assertParseFailure(parser, " r/001 from/2018/01/01 to/2018/01/02",
```
Comment (pikulet) [inline src/test/java/seedu/address/logic/parser/CheckoutCommandParserTest.java]: Since this parse failure is on the date format, the `ROOM_DESC_001` can still be used.

File: src/test/java/seedu/address/logic/parser/CheckinCommandParserTest.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    private CheckInCommandParser parser = new CheckInCommandParser();
+
+    @Test
+    public void parse_validArgs_returnsCheckInCommand() {
+        RoomNumber expectedRoomNumber = new RoomNumber(VALID_ROOM_NUMBER_001);
+        assertParseSuccess(parser, ROOM_DESC_001, new CheckInCommand(expectedRoomNumber));
+    }
+
+    @Test
+    public void parse_invalidArgs_throwsParseException() {
```
Comment (pikulet) [inline src/test/java/seedu/address/logic/parser/CheckinCommandParserTest.java]: Might want to add comments on what the test cases mean

File: src/test/java/seedu/address/logic/parser/CheckoutCommandParserTest.java (hunk 1 above)
Comment (adamwth) [reply src/test/java/seedu/address/logic/parser/CheckoutCommandParserTest.java]: okay will change so it makes it clearer which is the part that is supposed to cause the failure

File: src/test/java/seedu/address/logic/parser/CheckinCommandParserTest.java (hunk 2 above)
Comment (adamwth) [reply src/test/java/seedu/address/logic/parser/CheckinCommandParserTest.java]: noted. will do so in the next PR
